### PR TITLE
Clarify use of client for development use

### PR DESCRIPTION
**Pull request by frebib**
Regular users are not expected to need the provided client implementation and can instead go straight to their regular Matrix client and log in to use the sliding-sync proxy.

Fixes: https://github.com/matrix-org/sliding-sync/issues/28

*Rendered*: https://github.com/matrix-org/sliding-sync/blob/5735470e734388f2f96326bd01f3db0caba460c6/README.md

### CODE REVIEW

**Comment by frebib** [comment]
I'm not sure if this is direct enough. Let me know if anyone thinks it should be more clear

File: README.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ Using a Docker image:
 docker run --rm -e "SYNCV3_SERVER=https://matrix-client.matrix.org" -e "SYNCV3_SECRET=$(cat .secret)" -e "SYNCV3_BINDADDR=:8008" -e "SYNCV3_DB=user=$(whoami) dbname=syncv3 sslmode=disable host=host.docker.internal" -p 8008:8008 ghcr.io/matrix-org/sliding-sync:v0.98.0
 ```
 
-Then visit http://localhost:8008/client/ (with trailing slash) and paste in the `access_token` for any account on `-server`.
+If developing sliding-sync, a simple client is provided (not baked into the Docker image). Regular users may now log in with their sliding-sync compatible Matrix client.
+
+To use the stub client, visit http://localhost:8008/client/ (with trailing slash) and paste in the `access_token` for any account on `-server`.
```
Comment (DMRobertson) [inline README.md:46]: Drive-by: I'm guessing that `-server` was an old argument, used before everything was specified by env vars. And an additional cautionary remark.

```suggestion
To use the stub client, visit http://localhost:8008/client/ (with trailing slash) and paste in the `access_token` for any account on `SYNCV3_SERVER`. Note that this will consume to-device messages for the device associated with that access token.
```

**Comment by frebib** [comment]
I think GitHub is still a bit broken. I force-pushed a new commit but this PR isn't showing the change
https://github.com/frebib/sliding-sync/commit/f2803ac1ca40369a9a58d0b0d8a3f25c2e43409f

**Review by kegsay — APPROVED** [review]
Thanks!

**Comment by kegsay** [comment]
This needs sign-off though please @frebib

**Comment by frebib** [comment]
Sign-off how? The commit is already signed-off: https://github.com/matrix-org/sliding-sync/pull/40/commits/5735470e734388f2f96326bd01f3db0caba460c6

**Comment by DMRobertson** [comment]
> This needs sign-off though please @frebib

> Sign-off how? The commit is already signed-off: [5735470](https://github.com/matrix-org/sliding-sync/commit/5735470e734388f2f96326bd01f3db0caba460c6)

Good enough for me. Many thanks!
